book-network/src/main/java/com/alibou/book/book/Book.java

```diff
@@ -28,6 +29,7 @@ public class Book extends BaseEntity {
     private String title;
     private String authorName;
     private String isbn;
+    @Column(columnDefinition="TEXT")
```

Comment:
Using columnDefinition allows you to specify database-specific column types if needed, though it may reduce the portability of your application across different database systems.

You can keep the change in your code, for my case, I will keep it as 255 varchar.
Thank you for PR